src/utils/jsonabi.h

```diff
@@ -0,0 +1,199 @@
+#ifndef JSONABI_H
+#define JSONABI_H
+
+#include "../contract/customcontracts.h"
+#include "contractreflectioninterface.h"
+#include "utils.h" //nlohmann/json.hpp
+
+namespace JsonAbi {
```

Comment:
Missing Doxygen doc. From a quick glance I suppose it would be something like this?

```suggestion
/// Namespace for managing and converting contract ABI data to JSON format.
namespace JsonAbi {
```

Just an educated guess.